static/js/app.js

```diff
@@ -350,6 +350,43 @@ function watchForInstalledAddon() {
 }
 
 
+function showBannersIfNecessary() {
+  if (window.clientWidth < 750) {
+    return;
+  }
+  const dashboardBanners = document.querySelector(".dashboard-banners");
+  const bg = dashboardBanners.querySelector(".banner-gradient-bg");
```

Comment:
nit: Not sure if we should move this down closer to L367 to save an unnecessary DOM lookup if here are no `dashboardBanners` found or if the user isn't using Firefox or doesn't have the add-on installed.